capstone/src/main/java/com/google/sps/servlets/FileUnzipServlet.java

```diff
@@ -108,7 +108,9 @@ public static boolean analyzeApkFeatures(String nameOfApk, Blob blob, String use
           ze.getSize(),
           ze.getCompressedSize()
         );
-      
+        if (fileName.equals("AndroidManifest.xml")){
```

Comment:
I suspect we may want this to be a totally separate servlet if we end up wanting to do something like micro services but this is fine for now. You can call this locally via command line right?

Reply:
I thought I took it out, But it was initially there for debugging process